docs/src/development/troubleshoot.md

```diff
@@ -96,6 +96,15 @@ platform project:clear-build-cache -p <PROJECT_ID>
 
 The next build for each environment is likely to take longer as the cache rebuilds.
 
+## Access denied or Permission denied
+
+In most cases, issues accessing a project is caused by missing permissions for a given user.
+For more information see [how to manage user permissions](../administration/users.md).
```

Comment:
I'd consider leaving `how to` out of the link. Just like we leave it out of the titles of articles. Just a possibility.

Reply:
Good idea, removed them!